.planning/quick/260728-umo-admin-coffee-shop-photo-reorder-add-sort/PLAN.md

```
@@ -0,0 +1,31 @@
+---
+id: 260728-umo
+slug: admin-coffee-shop-photo-reorder-add-sort
+status: in-progress
+---
+
+# Quick: Admin coffee shop photo reorder
+
+## Goal
```

markdownlint-cli2 (0.23.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)